### PR TITLE
Add instructions for reverting from a split database to a single database implementation

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) adds a new topic with instructions for reverting from a split database to a single database implementation.

See internal build `128/guides/v2.4/config-guide/revert-split-database.html`.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/config-guide/revert-split-database.html
- https://devdocs.magento.com/guides/v2.3/config-guide/revert-split-database.html

whatsnew
Added a new topic with instructions for[ reverting from a split database to a single database ](https://devdocs.magento.com/guides/v2.4/config-guide/revert-split-database.html)implementation

### CODE REVIEW

File: src/guides/v2.3/config-guide/revert-split-database.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+## Verify your work
+
+To verify that your single database implementation is working properly, perform the following tasks and verify that data is added to the `magento_main` database tables using a database tool like [phpMyAdmin]({{ page.baseurl }}/install-gde/prereq/optional.html#install-optional-phpmyadmin):
+
+1. Verify that foreign keys have been restored. For example, the `QUOTE_STORE_ID_STORE_STORE_ID` key in the `quote` database table.
+1. Verify that customers can place orders from the storefront.
+1. Verify that orders created before reverting the split database to a single database are available in the Admin.
+
+{:.bs-callout-tip}
+After configuration, the single `magento_main` database should contain 435 tables.
```
Comment (keharper) [inline src/guides/v2.3/config-guide/revert-split-database.md:72]: This number (435) is not going to age well, and will likely vary, depending on the installed extensions. I recommend finding another way to visually confirm the procedure. Check that the migrated tables have content, maybe?

Comment (jeff-matthews) [reply src/guides/v2.3/config-guide/revert-split-database.md:72]: Good point @keharper. If @paliarush doesn't have a better recommendation, I'll just remove this line.

Comment (keharper) [reply src/guides/v2.3/config-guide/revert-split-database.md:72]: Oh. I see this is symlinked. The number won't be the same between releases anyway.

**Comment by jeff-matthews** [comment]
I was unable to rebase this branch onto master, so I created a new branch and did a cherry pick instead. See PR #8422.

File: src/guides/v2.3/config-guide/revert-split-database.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  - Setup
+---
+
+The [split database]({{ page.baseurl }}/config-guide/multi-master/multi-master.html) feature was deprecated in version 2.4.2 of {{ site.data.var.ee }} and {{ site.data.var.ce }}. Follow these instructions to revert from a split database to a single database implementation.
+
+## Revert a split database implementation
+
+Reverting from a split database to a single database implementation involves creating backups of the `magento_quote` and `magento_sales` databases before loading them into the single `magento_main` database.
+
+In this example, all three databases are installed on the same host named `magento2-mysql`.
```
Comment (paliarush) [inline src/guides/v2.3/config-guide/revert-split-database.md:17]: It is worth mentioning that in the examples below "root" MySQL user is used, but it has to be replaced with the appropriate user name for the Magento instance.

**Review by paliarush — APPROVED** [review]

**Pull request closed without merging**